assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -34,18 +34,18 @@ import {
   isKeyboardUser,
   KEYBOARD_USER_SELECTOR,
 } from '../../../utils/keyboardOnlyOutline';
+import useResizeEffect from '../../../utils/useResizeEffect';
 import { Dropdown as DropdownIcon } from '../../../icons';
 import Popup, { Placement } from '../../popup';
-import DropDownList from './list';
+import DropDownList, { DEFAULT_MIN_WIDTH } from './list';
 
 /* same min-width as ListContainer */
 const DropDownContainer = styled.div`
   display: flex;
   flex-direction: column;
-  min-width: 160px;
+  flex: 1;
```

Comment:
Setting `flex: 1` seemed to fix the issue, now the dropdown container expands with the space:
<img src="https://user-images.githubusercontent.com/40646372/94957370-7b9c8380-04a2-11eb-97ab-1076894924e9.png" height="300">